apps/int32_serial_publisher/prj.conf

```diff
@@ -0,0 +1,24 @@
+CONFIG_MAIN_STACK_SIZE=25000
+CONFIG_MAIN_THREAD_PRIORITY=3
+
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_NANO=n
+CONFIG_CPLUSPLUS=y
+CONFIG_LIB_CPLUSPLUS=y
+CONFIG_STD_CPP17=y
```

Comment:
Why do we need C++17? Some ROS package?

Reply:
It builds without this config flag, removed on all apps.